### ISSUE TITLE
attemptsTable not using displaystyle formatting

### ISSUE THREAD

**Issue by somiaj** [openwebwork/webwork2#1921]
I noticed that the attempts table preview and correct answer columns don't seem to use displaymode any longer (at least consistently). Using `Compute('x/3')` as my answer gives the following table:

![image](https://user-images.githubusercontent.com/18082412/227758230-f2528162-9218-4ab8-b09c-a2b76613af77.png)

In looking into this I noticed that the change to mojolicious changed:

```perl
return '<script type="math/tex; mode=display">' . $tex . '</script>';
```

to

```perl
return $self->c->tag('script', type => 'math/tex', mode => 'display', $self->c->b($tex));
```

which took the `mode=display` option and moved it to its own tag. If I move it back, I seem to get displaystyle working on the two columns again.

```perl
return $self->c->tag('script', type => 'math/tex; mode=display', $self->c->b($tex));
```

![image](https://user-images.githubusercontent.com/18082412/227758306-bede70c7-e6de-43a5-8c3b-2b529b405639.png)

**Comment by drgrice1** [comment]
That is an obvious mistake.  Can you put in a pull request?

**Comment by drgrice1** [comment]
Actually, I have a couple of other changes to the attempts table in a branch already.  I will add this, and put it all in together.

**Comment by pstaabp** [comment]
fixed in #1924
